Headers/Public/VLCLibrary.h

```diff
@@ -130,8 +130,8 @@ NS_ASSUME_NONNULL_BEGIN
 @required
 /**
  * called when VLC wants to print a debug message
- * \param error the dialog title
  * \param message the error message
+ * \param level the debug level
```

Comment:
could you also change the error to debug for param message ?

Reply:
I'm not sure to understand what you mean here?

Reply:
```param message the error message``` should be the debug message I think

Reply:
Ah indeed, I will change this!